README.md

```diff
@@ -1,101 +1,38 @@
-# HTML & CSS3 & JavaScript Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+## **TODO LIST**
+A simple app to help you manage the things you have to do in a day.<br />
 
-## Set-up GitHub Actions
+## Built With
 
-Please do the following **steps in this order**:
+- HTML,CSS, JavaScript
+- Frameworks, Webpack
+- Github, Linters
 
```

Comment:
- [ ] [OPTIONAL] Your readme is great :+1:  but I would like to recommend you to add the `Getting started` section to your readme because we read a project’s README in order to figure out how to start using a project. This section should be the bulk of your README contents. this section is to help someone who has never used your project get set up, and write down the exact instructions. :smile:

Reply:
@ Oooh okay noted, thanks